### PR TITLE
Update prepare-deployment.md

### PR DESCRIPTION
**Pull request by msbemba**
updated basic permissions in the document

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7314

File: microsoft-365/security/defender-endpoint/prepare-deployment.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Microsoft recommends using [Privileged Identity Management](/azure/active-direct
 
 Defender for Endpoint supports two ways to manage permissions:
 
-- **Basic permissions management**: Set permissions to either full access or read-only. In the case of basic permissions management users with Globa Administrator or Security Administrator role in Azure Active Directory have full access while the Security reader role has read-only access.
+- **Basic permissions management**: Set permissions to either full access or read-only. In the case of basic permissions management users with Globa Administrator or Security Administrator role in Azure Active Directory have full access. While the Security reader role has read-only access, it does not grant you access to  view machines/ device inventory.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/prepare-deployment.md]:
```suggestion
- **Basic permissions management**: Set permissions to either full access or read-only. In the case of basic permissions management users with Globa Administrator or Security Administrator roles in Azure Active Directory have full access. While the Security reader role has read-only access, it does not grant you access to view machines/device inventory.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**